test/unit/api/compress/compress_store.cpp

```diff
@@ -0,0 +1,11 @@
+//==================================================================================================
+/**
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+**/
+//==================================================================================================
+
+#include "test.hpp"
+
```

Comment:
why the empty file then ?

Reply:
I figured so that when one comes looking, the find this

Reply:
well it's an empty target that just consume test compilation time no ?

Reply:
Agreed that I'll delete it in the next patch